ChangeLog

```diff
@@ -1,5 +1,16 @@
 For detailed information about the changes below, please see the git log
-or visit: http://log.libusb.info
+or visit: https://log.libusb.info
+
+202?-??-??: v1.0.30
+* Added hotplug support on Microsoft Windows.
+* Reintroduced RAW_IO support in WinUSB backend.
+* Worked around a macOS 26 Tahoe compatibility breakage due to Apple changing kUSBHostPortPropertyPortNumber.
+* Added new API libusb_get_device_string() to access device strings without opening the device.
+* Added new API libusb_get_session_data() which returns the OS-specific handle.
+* LIBUSB_API_VERSION bumped to 0x0100010C for the new functions in 1.0.30.- On Android, fixed intermittent failures in get_usbfs_fd().
+* On Windows, fixed bus number assignment for root hub device, preventing duplicate bus number assignments.
+* Fixed compilation with Microsoft Visual Studio 2026.
+* Fixed various compiler warnings, improved tests and examples.
```

Comment:
Stylistic nits: Like for commit, the imperative form is preferred. Looking down the list, it has always been like this.